docs/source/workflow_integration/databricks.rst

```diff
@@ -0,0 +1,10 @@
+
+Databricks integration
+=============================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Databricks
+
+   databricks_workspace.md
+   visualisation.md
```

Comment:
May I suggest to convert this to MyST? 😄 

````suggestion
# Databricks integration

```{toctree}
:caption: Databricks
:maxdepth: 2

databricks_workspace.md
visualisation.md
```
````

Reply:
@stichbury kindly contributed this. I'm actually not familiar with MyST, could you explain the advantage of making this change?

Reply:
Let the machine do all the hard work 🤖 https://github.com/executablebooks/rst-to-myst

Reply:
Made this change to `databricks.rst`.